src/Features/Units/Converter/data_types/_visibility.ts

```diff
@@ -0,0 +1,14 @@
+import { UnitSystem } from "../../types"
+import { DataTypeConversion } from "../conversions"
+
+export class Visibility extends DataTypeConversion {
+  public convertTo(value: number, unitSystem: UnitSystem): number {
+    switch (unitSystem) {
+      case UnitSystem.mariners:
+        return value / 1852
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 1852](https://app.codacy.com/app/gmri/Neracoos-1-Buoy-App/pullRequest?prid=4354739)